src/app/Library/CrudPanel/ViewNamespaces.php

```diff
@@ -1,22 +1,22 @@
 <?php
 
-namespace Backpack\CRUD\app\Library\CrudPanel\Traits;
+namespace Backpack\CRUD\app\Library\CrudPanel;
 
-trait HasViewNamespaces
+class ViewNamespaces
 {
-    private $viewNamespaces = [];
+    private static $viewNamespaces = [];
 
     /**
      * Return all the view namespaces including the ones stored in the laravel config files.
      *
      * @param  string  $domain  (eg. fields, filters, buttons)
      * @return array
      */
-    public function getViewNamespacesFor(string $domain)
+    public static function getViewNamespacesFor(string $domain)
```

Comment:
Now that it's a class, and you call it `ViewNamespaces::smth()`, I think it's odd to keep the same method names. How about...

```diff
- ViewNamespaces::getViewNamespacesFor($domain);
+ ViewNamespaces::getFor($domain);

- ViewNamespaces::addViewNamespacesFor($domain, $viewNamespaces);
+ ViewNamespaces::addFor($domain, $viewNamespaces);
```

We could make 'em polymorphic, but if that's messy, we could also do...

```php
ViewNamespaces::getOneFor($domain);
ViewNamespaces::addOneFor($domain);
ViewNamespaces::getMultipleFor($domain);
ViewNamespaces::addMultipleFor($domain);

// or just
ViewNamespaces::getOne($domain);
ViewNamespaces::addOne($domain);
ViewNamespaces::getMultiple($domain);
ViewNamespaces::addMultipleFor($domain);
```

Damn... naming this won't be easy. Any other ideas @pxpm ?